immuni_exposure_ingestion/apis/ingestion.py

```diff
@@ -89,6 +89,10 @@
 @validate(
     location=Location.JSON,
     province=Province(),
+    country_of_interest=fields.List(
```

Comment:
If it's a list, I'd go for `countries_of_interest`. 🙏

Reply:
changed, you're right!